src/features/events_list/components/EventCard/EventCard.tsx

```
@@ -86,7 +92,10 @@ export function EventCard({
       ) : null}
 
       <div className={s.footer}>
-        <Text type="caption">{i18n.t('updated') + ` ${formattedTime}`}</Text>
+        <div className={s.timeInfo}>
+          <Text type="caption">{i18n.t('started') + ` ${formattedStartedAt}`}</Text>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)